docs(WebGL API): enhance createShader(), compileShader(), and shaderSource() methods (#38008)
mdn/content: files/en-us/web/api/webglrenderingcontext/createshader/index.md

### Parameters

- `type`
  - : Either `gl.VERTEX_SHADER` or `gl.FRAGMENT_SHADER`. The {{domxref("WebGLRenderingContext")}} will set the `gl.INVALID_ENUM` error flag if an unacceptable value has been specified.

### Return value

> A new ({{domxref("WebGLShader")}}), or `null` if an error occurs creating the ({{domxref("WebGLShader")}}) (such as providing an invalid value for `type`).

Reviewer: ```suggestion
A new {{domxref("WebGLShader")}} instance, or `null` if an error occurs creating the shader (for example, because `type` was an invalid value).
```
